subscription-service/src/test/kotlin/com/egm/stellio/subscription/service/SubscriptionServiceTests.kt

```diff
@@ -899,4 +899,90 @@ class SubscriptionServiceTests : WithTimescaleContainer {
 
         assertEquals(res, true)
     }
+
+    /**
+     * Test for function 'getAllSubscriptions' with two subscription.
+     * One with the 'lastNotification' attribute set to null and the other with the 'lastNotification' now and
+     * 'timeInterval' set to 1
+     *
+     * @return list of subscription
+     */
+    @Test
+    fun `it should return all Entity where 'timeInterval' are expired `() {
+
+        val persistedSubscription = subscriptionService.getById(subscription1Id).block()!!
+        val notification = Notification(subscriptionId = subscription1Id, data = emptyList())
+
+        val updateResult =
+            subscriptionService.updateSubscriptionNotification(persistedSubscription, notification, true)
+                .then(subscriptionService.getById(subscription1Id))
+
+        StepVerifier.create(updateResult)
+            .expectNextMatches {
+                it.id == subscription1Id &&
+                    it.notification.status == StatusType.OK &&
+                    it.notification.timesSent == 1 &&
+                    it.notification.lastNotification != null &&
+                    it.notification.lastSuccess != null &&
+                    it.notification.lastFailure == null
+            }
```

Comment:
you don't really care about this (I guess this is already tested somewhere in this class). what you want to test here is that you get the expected subscriptions:
- never notified and time interval reached compared to creation time of the subscription
- already notified and time interval reached since the last notification

Reply:
Ok I changed that